File: Latihan_a/Processor.cpp
#include <bits/stdc++.h>
#include <string>

using namespace std;

class Processor {

    private:
        // atribut
        string name;
        int price;

    public:
        // konstruktor
        Processor(){}

        Processor(string name, int price)
        {
            this->name = name;
            this->price = price;
        }

        // setter
        void setName(string name)
        {
            this->name = name;
        }

        void setPrice(int price)
        {
            this->price = price;
        }

        // getter
        string getName()
        {
            return this->name;
        }

        int getPrice()
        {
            return this->price;
        }

        // destruktor
        ~Processor(){}
};

File: Latihan_b/Php/Processor.php
<?php

class Processor {

    // atribut
    private $name;
    private $price;

    
    // konstruktor
    function __construct($name = "", $price = 0)
    {
        $this->name = $name;
        $this->price = $price;
    }

    // setter
    function setName($name)
    {
        $this->name = $name;
    }

    function setPrice($price)
    {
        $this->price = $price;
    }

    // getter
    function getName()
    {
        return $this->name;
    }

    function getPrice()
    {
        return $this->price;
    }
}

?>